spec/services/nginx_spec.rb

```diff
@@ -3,11 +3,27 @@
 require 'spec_helper'
 set :os, family: 'redhat', release: '9', arch: 'x86_64'
 
-describe package('nginx') do
-  it { should be_installed }
-end
+describe 'Checking nginx' do
+  describe package('nginx') do
+    it { should be_installed }
+  end
+
+  describe service('nginx') do
+    it { should be_enabled }
+    it { should be_running }
+  end
+
+  describe port(443) do
+    it { should be_listening }
+  end
```

Comment:
It can be listening or it not, this should be checked only is the node has ngninx